src/main/java/edu/kis/powp/jobs2d/features/CommandParser.java

```diff
@@ -0,0 +1,97 @@
+package edu.kis.powp.jobs2d.features;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+import edu.kis.powp.jobs2d.drivers.adapter.JsonAdapter;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class CommandParser {
+    private File file;
+    private List<DriverCommand> commandList;
+
+    public CommandParser(File file) {
+        this.file = file;
+        this.commandList = new ArrayList<>();
+    }
+
+    public void fillListFromFile() throws Exception {
+        String fileName = this.file.getName();
+        String fileType = fileName.substring(fileName.lastIndexOf('.') + 1).toLowerCase();
+
+        switch (fileType) {
+            case "txt":
```

Comment:
please use constants or enum to make the meaning of those strings clearer.

Reply:
Constatnts used